atmo/jobs/models.py

```diff
@@ -204,14 +187,18 @@ def run(self):
             is_public=self.is_public,
             job_timeout=self.job_timeout,
         )
-        self.last_run_date = timezone.now()
-        self.update_status()
-        self.save()
+        # Create new job history record.
+        run = self.runs.create(
+            spark_job=self,
+            jobflow_id=jobflow_id,
+            scheduled_date = timezone.now(),
+        )
+        run.update_status()
```

Comment:
You can do that by calling `delattr(self, 'latest_run')` or `del self.latest_run`. It may raise an AttributeError though if it wasn't in fact populated before, so wrap it in a try: except block.